index.html

```diff
@@ -185,9 +185,9 @@ <h3 class="let">LET'S CONNECT</h3>
         </section>
 
         <section id="Contact" class="contact-section">
-            <img src="image-source/Ellipse 12.png" class="ellipse-12 el" alt="Ellipse 12">
-            <img src="image-source/Ellipse 33.png" class="ellipse-33 el" alt="Ellipse 33">
-            <img src="image-source/Rectangle 4.png" class="rectangle-4 el" alt="Rectangle 4">
+            <img src="image-source/Ellipse 12.png" class="ellipse-12 el" alt="">
+            <img src="image-source/Ellipse 33.png" class="ellipse-33 el" alt="">
+            <img src="image-source/Rectangle 4.png" class="rectangle-4 el" alt="">
```

Comment:
- [ ] Great job on adding the alt attribute here 🎉 it really makes your page more accessible. In some cases, it's ok to leave it empty and you did that well in some others but in this case, it's necessary to tell the reader that this is a  logo of ..., in case it's not rendered. 🤓